From 0956e61c7c8653218bcaa46087d4508fc795feaa Mon Sep 17 00:00:00 2001
From: Peter Eckersley <pde@users.noreply.github.com>
Date: Wed, 30 Nov 2016 10:47:10 -0800
Subject: [PATCH] Remove the sphinxcontrib.programout [docs]dependency (#3830)

- By making a static text file at release time, and including that
   instead.
---
 docs/conf.py      | 1 -
 docs/jws-help.txt | 8 ++++++++
 docs/man/jws.rst  | 2 +-
 setup.py          | 1 -
 4 files changed, 10 insertions(+), 10 deletions(-)
 create mode 100644 acme/docs/jws-help.txt

diff --git a/docs/conf.py b/docs/conf.py
index 55f5eee..dea23a8 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -39,7 +39,6 @@
     'sphinx.ext.todo',
     'sphinx.ext.coverage',
     'sphinx.ext.viewcode',
-    'sphinxcontrib.programoutput',
 ]
 
 autodoc_member_order = 'bysource'
diff --git a/docs/jws-help.txt b/docs/jws-help.txt
new file mode 100644
index 0000000..34cf5ce
--- /dev/null
+++ b/docs/jws-help.txt
@@ -0,0 +1,8 @@
+usage: jws [-h] [--compact] {sign,verify} ...
+
+positional arguments:
+  {sign,verify}
+
+optional arguments:
+  -h, --help     show this help message and exit
+  --compact
diff --git a/docs/man/jws.rst b/docs/man/jws.rst
index fb3121a..d7ff8f4 100644
--- a/docs/man/jws.rst
+++ b/docs/man/jws.rst
@@ -1 +1 @@
-.. program-output:: jws --help all
+.. literalinclude:: ../jws-help.txt
diff --git a/setup.py b/setup.py
index 2b32f7e..895889c 100644
--- a/setup.py
+++ b/setup.py
@@ -49,7 +49,6 @@
 docs_extras = [
     'Sphinx>=1.0',  # autodoc_member_order = 'bysource', autodoc_default_flags
     'sphinx_rtd_theme',
-    'sphinxcontrib-programoutput',
 ]
 
 
